pkgs/servers/janusgraph/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, stdenv, fetchzip, adoptopenjdk-jre-openj9-bin-11, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "janusgraph";
+  version = "0.6.2";
+
+  src = fetchzip {
+    url = "https://github.com/JanusGraph/janusgraph/releases/download/v${version}/janusgraph-${version}.zip";
+    sha256 = "sha256-8TMYk8gGyL71zcFk0Lgo7Isvm4k3eh/H6PjfVePpkI4=";
+  };
+
+  buildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin $out/share/java
+    install -D $src/lib/*.jar $out/share/java
+    cd $src
+    find conf scripts -type f -exec install -D {} $out/share/{} \;
+
+    JANUSGRAPH_LIB=$out/share/java
+    CP=""
+    # Add the slf4j-log4j12 binding
+    CP="$CP":$(find -L $JANUSGRAPH_LIB -name 'slf4j-log4j12*.jar' | sort | tr '\n' ':')
+    # Add the jars in $JANUSGRAPH_HOME/lib that start with "janusgraph"
+    CP="$CP":$(find -L $JANUSGRAPH_LIB -name 'janusgraph*.jar' | sort | tr '\n' ':')
+    # Add the remaining jars in $JANUSGRAPH_HOME/lib.
+    CP="$CP":$(find -L $JANUSGRAPH_LIB -name '*.jar' \
+                    \! -name 'janusgraph*' \
+                    \! -name 'slf4j-log4j12*.jar' | sort | tr '\n' ':')
+
+    makeWrapper ${adoptopenjdk-jre-openj9-bin-11}/bin/java $out/bin/janusgraph-server \
```

Comment:
Can we use a java higher up in the wrapper chain?

Reply:
I've simplified it to `jdk11`.

Reply:
thats good enough